tests/test_util/test_util_rst.py

```diff
@@ -15,7 +15,7 @@ def test_escape():
 
 
 @pytest.mark.sphinx('html', testroot='root')
```

Comment:
Heya you can remove these as well

Reply:
Happy to. Can you please give me a brief explanation so that I can make a useful commit message / to improve my understanding?

Reply:
Thank you, I have made this change.